frontend/packages/core/src/navigation.tsx

```diff
@@ -8,7 +8,7 @@ import {
 } from "react-router-dom";
 import type { PartialPath, State } from "history";
 
-const UTM_PARAMS = ["utm_source", "utm_medium"];
+const UTM_PARAMS = ["utm_source", "utm_medium", "utm_campaign", "utm_term", "utm_content"];
```

Comment:
It seems like there [five different UTM parameters](https://en.wikipedia.org/wiki/UTM_parameters).

Reply:
yeah, good to expand, but we only use `utm_source` and `utm_medium`.